docs/xcm/building-with-xcm/xc-reserve-transfer.md

```diff
@@ -47,12 +46,16 @@ This functionality is exposed to EVM smart contracts via precompiles. Interface
 
 **Note:** there is another version of `assets_reserve_transfer` precompile that accepts `address` instead of `bytes32` for `recipient_account_id`.
 
+# Transferring native assets
+
+Current API identifes assets being transferred by specifying an H160 style address (XC20). This prevents us from sending native token since there's no representation for it. However, there is a workaround for that which uses EVM `msg.value` API.
+
+Precompile implemenation checks `msg.value` and, if positive, treats it as another asset to be sent (`MultiLocation { parents: 0, interior: Here }`). In that case, native asset is added to the tail of `asset_id` and `asset_amount` lists and can be indexed by `fee_index` as any other asset in the list. Its value would be set equal to `msg.value`.
+
```

Comment:
Would be good to have an example of this. E.g. both arrays for assets and amounts params are empty, but `.value` is greater than 0 which results in native currency transfer.